lib/reek/smells/attribute.rb

```diff
@@ -1,4 +1,6 @@
+require_relative 'smell_configuration'
```

Comment:
smell_detector already requires smell_configuration, so this one is not needed per se.